src/server/config/config.service.ts

```diff
@@ -201,21 +202,22 @@ class ConfigService {
    * NOTE: This is needed to properly connect to redis over TLS
    */
 
-  public get redisURL(): string {
+  public get redisClientOptions(): ClientOpts {
     const {
       REDIS_HOST,
       REDIS_PORT,
       REDIS_PASSWORD,
-      NODE_ENV,
     } = this.env;
-    const redis = new URL(`redis://${REDIS_HOST}:${REDIS_PORT}`);
-    if (NODE_ENV === 'production') {
-      redis.protocol = 'rediss:';
-    }
+    const redis = new URL(`rediss://${REDIS_HOST}:${REDIS_PORT}`);
     if (REDIS_PASSWORD) {
       redis.password = REDIS_PASSWORD;
     }
-    return redis.toString();
```

Comment:
Minor point but the docblock for this getter says: _"Return the redis connection string"_. Since it's returning `ClientOpts` this might want updating..

Reply:
Still worth changing. Did that in 5d9ca3e